src/main/java/frc/robot/commands/InterstellarAccuracy.java

```diff
@@ -0,0 +1,141 @@
+package frc.robot.commands;
+
+import java.util.List;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.FieldMapHome;
+import frc.robot.subsystems.Carousel;
+import frc.robot.subsystems.DriveTrain;
+import frc.robot.subsystems.Shooter;
+public class InterstellarAccuracy extends SequentialCommandGroup implements AutoCommandInterface {
+    Rotation2d rotation180 = Rotation2d.fromDegrees(180.0);
+    Rotation2d rotation270 = Rotation2d.fromDegrees(270.0);
+    Rotation2d rotation90 = Rotation2d.fromDegrees(90.0);
```

Comment:
these ought to be "final", and I believe 90 and 270 are not used?

Reply:
Correct.

Reply:
Fixed.